Reactions/reactor.H

```diff
@@ -141,20 +152,20 @@ void reactor_close();
 
   int NNZ;
   SUNMatrix* PS;
-  realtype** Jdata = NULL;
+  amrex::Real** Jdata = NULL;
   int** rowVals = NULL;
   int** rowPtrs = NULL;
   int** colPtrs = NULL;
   int** colVals = NULL;
   int* indx = NULL;
-  realtype** JSPSmat = NULL;
+  amrex::Real** JSPSmat = NULL;
 #ifdef USE_KLU_PP
   klu_common* Common;
   klu_symbolic** Symbolic;
   klu_numeric** Numeric;
 #else
-  realtype**(**Jbd);
-  realtype**(**P);
+  amrex::Real**(**Jbd);
```

Comment:
Let's also remove the parentheses on these statements since they generate compiler warnings.

Reply:
@esclapez - these are used in cvode cpu case. I am still baffled why you need a 4d pointer for a 2d matrix?
likewise, I think this should be related to a different PR as this is part of cvode.

Reply:
I have no idea. But that's the form Cvode is expecting ... Probably help minimize the storage of the sparse matrix.